### ISSUE TITLE
OpenHAB offline, when trying to access from myOpenHAB.org instance

### ISSUE THREAD

**Issue by demichve** [openhab/openhab-cloud#459]
As of last night (26th May), you cannot access anymore your local OpenHAB instance using web browser, when using myopenHAB.org service.

I am not sure, if this is the right place to raise this issue.

There are already discussions in community:
https://community.openhab.org/t/openhab-is-offline-error/154427/21
https://community.openhab.org/t/cloud-connector-ipv6-errors/156209

Steps to reproduce:
Run https://myopenhab.org/ and login with your user credentials.
Click on the dashboard link -> fails with "OpenHAb Offline"

Notifications are sent using the service, the rest is dead.

Michael

**Comment by curlyel** [comment]
In my case, myOpenHAB considers my openHAB server to be online:
![image](https://github.com/openhab/openhab-cloud/assets/11849305/7cd672dc-d188-4aca-b2e3-6c02b0ff2f9c)

And: remote integration via Amazon Alexa is still working. Just the MainUI is not shown.
See also for more details: https://github.com/openhab/openhab-addons/issues/16809

**Comment by digitaldan** [comment]
Sorry about that, we did a small deployment that created an api endpoint to return some info on the proxy settings for our mobile clients. Unfortunately this set the host header we send in the request to a users’s openHAB to https://home.myopenhab.org:443 instead of home.myopenhab.org:443 (its not suppose to have the protocol in it). Jetty, which is what openHAB uses for web requests, choked on this and was trying to parse it as a ipv6 address. So while openHABs where connected, http requests back to them will malformed.

I just made a fix and deployed it, things should be back to normal now or at least in a few mins when everyone connects.
